main/src/library/Down.flix

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright 2022 Nina Andrup Pedersen
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+pub enum Down[a] {
```

Comment:
Should it have some kind of description?

Reply:
Yes, take a look at:

https://hackage.haskell.org/package/base-4.17.0.0/docs/Data-Ord.html#t:Down

for inspiration.